site2/docs/administration-stats.md

```diff
@@ -4,60 +4,167 @@ title: Pulsar stats
+| earliestMsgPublishTimeInBacklogs                 | The publish time of the earliest message in the backlog (in milliseconds).                               |
+| bytesInCounter                                   | The total bytes published to the topic.                                                                  |
+| msgInCounter                                     | The total messages published to the topic.                                                               |
+| bytesOutCounter                                  | The total bytes delivered to consumers.                                                                  |
+| msgOutCounter                                    | The total messages delivered to consumers.                                                               |
+| msgChunkPublished                                | The topics that have chunked messages published on it.                                                   |
+| backlogSize                                      | The estimated total unconsumed or backlog size (in bytes).                                               |
+| offloadedStorageSize                             | Space that is used to store the offloaded messages for the topic (in bytes).                             |
+| waitingPublishers                                | The number of publishers waiting in a queue in exclusive access mode.                                    |
+| deduplicationStatus                              | The status of message deduplication for the topic.                                                       |
+| topicEpoch                                       | The topic epoch or empty if not set.                                                                     |
+| filteredEntriesCount                             | The count of skipped entries for the topic.                                                              |
+| nonContiguousDeletedMessagesRanges               | The number of non-contiguous deleted messages ranges.                                                    |
+| nonContiguousDeletedMessagesRangesSerializedSize | The serialized size of non-contiguous deleted messages ranges.                                           |
+| ownerBroker                                      | The broker that owns this topic.                                                                         |
+| [publishers](#producer-stats)                    | The list of all local publishers on the topic, ranging from zero to thousands.                           |
+| [subscriptions](#subscription-stats)             | The list of all local subscriptions to the topic.                                                        |
+| [replication](#replication-stats)                                      | This section gives the stats for cross-colo replication of this topic.                                   |
```

Comment:
Do we need to align the table?

Reply:
Done:) The table has been auto-formatted.